### PR TITLE
Unified README added

### PR DESCRIPTION
**Pull request by 650elx**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -134,37 +122,6 @@ except NumbersException as err:
...
-If all logging returned by this SDK needs to be disabled, usage of `NullHanlder` provided by the standard `logging` module is advised.  
-
-
-## Custom HTTP client implementation
-
-By default, two HTTP implementations are provided:
-- Synchronous using `requests` HTTP library
-- Asynchronous using `aiohttp` HTTP library
-
-For creating custom HTTP client code, use either `Client` or `ClientAsync` client and inject your transport during initialisation:
```
Comment (Dovchik) [inline README.md:154]: It's nice to have example, maybe you can keep them?

File: README.md [hunk 2]
```diff
@@ -1,43 +1,29 @@
-<h1 align="center">
+# Sinch Python SDK
 
-   [![Sinch Logo](https://developers.sinch.com/static/logo-07afe977d6d9dcd21b066d1612978e5c.svg)](https://www.sinch.com)
```
[the remark this answers is not in the document]
Comment (650elx) [reply README.md]: I've tried it but it looks mediocre IMO. I prefer to keep this as "center".

File: README.md [hunk 3]
```diff
@@ -1,43 +1,29 @@
...
-   Python SDK
+To use Sinch services, you'll need a Sinch account and access keys. You can sign up for an account and create access keys at [dashboard.sinch.com](https://dashboard.sinch.com).
 
-   [![License](https://img.shields.io/badge/License-Apache_2.0-blue.svg)](https://github.com/sinch/sinch-sdk-python/blob/main/LICENSE)
-
-
-   [![Python 3.9](https://img.shields.io/badge/python-3.9-blue.svg)](https://www.python.org/downloads/release/python-390/)
-   [![Python 3.10](https://img.shields.io/badge/python-3.10-blue.svg)](https://www.python.org/downloads/release/python-3100/)
-   [![Python 3.11](https://img.shields.io/badge/python-3.11-blue.svg)](https://www.python.org/downloads/release/python-3110/)
-   [![Python 3.12](https://img.shields.io/badge/python-3.12-blue.svg)](https://www.python.org/downloads/release/python-3120/)
```
[the remark this answers is not in the document]
Comment (650elx) [reply README.md]: I'm happy to bring this stuff back :)

File: README.md (hunk 1 above)
Comment (650elx) [reply README.md]: done

File: README.md [hunk 4]
```diff
@@ -1,5 +1,23 @@
+<h1 align="center">
```
Comment (Dovchik) [inline README.md:1]: In my case, Nuget didn't support `<h1>` tag in it's markdown language renderer so I was forced to remove it. 
Additionally, layout of Sinch is left aligned so maybe we can follow same guideline here?

**Review by Dovchik — APPROVED** [review]

**Pull request merged**